app/(main)/academic-detail/[year]/[semester]/page.tsx

```
@@ -0,0 +1,27 @@
+export const fetchCache = 'force-cache';
+
+export default async function AcademicDetailPage({params}: {params: Promise<{year: number, semester: number}>}) {
+  const {year, semester} = await params;
+
+  const [semesters, data] = await Promise.all([
+    getSemesters(),
+    getAcademicDetail(year, semester),
+  ]);
+  
+  return (
+    <Suspense fallback={<div></div>}>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)